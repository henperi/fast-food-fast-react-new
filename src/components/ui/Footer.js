import React from "react";
import { CartSlider1 } from "./CartSlider";

const Footer = () => (
  <div>
    <CartSlider1 />
    <footer className="footer text-center">
      <div>Fast Food Fast. &copy; All Rights Reserved</div>
    </footer>
  </div>
);

export default Footer;
