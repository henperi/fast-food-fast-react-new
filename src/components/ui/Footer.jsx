import React from 'react';
import { CartSlider1 } from './CartSlider';

const Footer = props => (
  <div>
    <CartSlider1 {...props} />
    <footer className="footer text-center">
      <div>Fast Food Fast. &copy; All Rights Reserved</div>
    </footer>
  </div>
);

export default Footer;
