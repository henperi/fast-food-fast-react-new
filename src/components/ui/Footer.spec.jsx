import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

test('should render Header correctly', () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});
