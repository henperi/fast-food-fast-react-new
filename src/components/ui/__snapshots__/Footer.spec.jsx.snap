// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <Connect(CartSlider1) />
  <footer
    className="footer text-center"
  >
    <div>
      Fast Food Fast. Â© All Rights Reserved
    </div>
  </footer>
</div>
`;
